cypress/e2e/tests/migration/applicationinventory/applications/sort.test.ts

```diff
@@ -74,6 +74,24 @@ describe(["@tier3"], "Application inventory sort validations", function () {
         verifySortDesc(afterDescSortList, unsortedList);
     });
 
+    it("Analysis status sort validations", function () {
+        Application.open();
+        // get unsorted list when page loads
+        const unsortedList = getTableColumnData(analysis);
+        // Sort the application inventory by Tag count in ascending order
+        clickOnSortButton(analysis, SortType.ascending);
+        cy.wait(2 * SEC);
+        // Verify that the application inventory table rows are displayed in ascending order
+        const afterAscSortList = getTableColumnData(analysis);
+        verifySortAsc(afterAscSortList, unsortedList);
+        // Sort the application inventory by analysis in descending order
+        clickOnSortButton(analysis, SortType.descending);
+        cy.wait(2000);
+        // Verify that the application inventory table rows are displayed in descending order
```

Comment:
The code is explanatory enough so we can remove the comments.

I think the `cy.wait` lineas can be removed as well

Reply:
There are methods like verifySortAsc and verifySortDesc in utils which can be used directly .Please check.

Reply:
used these methods here a well, 
the verifySortDesc is only a verification function that checks that the filter works right, 
after the application filters the apps by pressing on the filter button